rasa/shared/importers/importer.py

```diff
@@ -496,7 +496,9 @@ async def _additional_training_data_from_stories(self) -> TrainingData:
         sorted_utterances = sorted(
             utterances, key=lambda user: user.intent_name or user.text
         )
-        sorted_actions = sorted(actions, key=lambda action: action.action_name)
+        sorted_actions = sorted(
+            actions, key=lambda action: action.action_name or action.action_text
```

Comment:
but an action name can't be `None` or `''` can it (unlike the intent name)?

Reply:
it will actually once we add end-to-end reading 👍 (see https://github.com/RasaHQ/rasa/pull/6605)